### ISSUE TITLE
Cannot install dplyr in R context

### ISSUE THREAD

**Issue by sivakon** [oracle/graal#327]
I am trying to install `dplyr` in R context. I get the following error.

```
Error: package or namespace load failed for ‘rlang’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/library/rlang/libs/rlang.so'
  dlopen(/Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/library/rlang/libs/rlang.so, 6): Symbol not found: _Rf_any_duplicated3
  Referenced from: /Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/library/rlang/libs/rlang.so
  Expected in: flat namespace
 in /Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/library/rlang/libs/rlang.so
Error: loading failed
```

The complete stack trace:
```
The downloaded source packages are in
	‘/private/var/folders/vx/_5fgc0ps1_s46k0jyf4fw2w5gn8nsx/T/RtmpegOTl6/downloaded_packages’
Updating HTML index of packages in '.Library'
Making 'packages.html' ... done
Warning message:
In install.packages("rlang", repos = "http://cran.us.r-project.org") :
  installation of package ‘rlang’ had non-zero exit status
#object[org.graalvm.polyglot.Value 0x1dc9c4ce "NULL"]

graal-test.core=> (.eval context "R" "
             #_=> install.packages('dplyr', repos='http://cran.us.r-project.org')")
Content type 'application/x-gzip' length 323899 bytes (316 KB)
* installing *source* package ‘rlang’ ...
** package ‘rlang’ successfully unpacked and MD5 sums checked
** libs
gcc -I/Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/include -DNDEBUG -I./lib/  -O2 -I/cm/shared/apps/pcre/8.38/include -I/cm/shared/apps/bzip2/1.0.6/include -I/cm/shared/apps/xz/5.2.2/include -I/cm/shared/apps/curl/7.50.1/include -DFASTR   -fPIC  -O2 -I/cm/shared/apps/pcre/8.38/include -I/cm/shared/apps/bzip2/1.0.6/include -I/cm/shared/apps/xz/5.2.2/include -I/cm/shared/apps/curl/7.50.1/include  -DFASTR -c capture.c -o capture.o
gcc -I/Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/include -DNDEBUG -I./lib/  -O2 -I/cm/shared/apps/pcre/8.38/include -I/cm/shared/apps/bzip2/1.0.6/include -I/cm/shared/apps/xz/5.2.2/include -I/cm/shared/apps/curl/7.50.1/include -DFASTR   -fPIC  -O2 -I/cm/shared/apps/pcre/8.38/include -I/cm/shared/apps/bzip2/1.0.6/include -I/cm/shared/apps/xz/5.2.2/include -I/cm/shared/apps/curl/7.50.1/include  -DFASTR -c export.c -o export.o
gcc -I/Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/include -DNDEBUG -I./lib/  -O2 -I/cm/shared/apps/pcre/8.38/include -I/cm/shared/apps/bzip2/1.0.6/include -I/cm/shared/apps/xz/5.2.2/include -I/cm/shared/apps/curl/7.50.1/include -DFASTR   -fPIC  -O2 -I/cm/shared/apps/pcre/8.38/include -I/cm/shared/apps/bzip2/1.0.6/include -I/cm/shared/apps/xz/5.2.2/include -I/cm/shared/apps/curl/7.50.1/include  -DFASTR -c internal.c -o internal.o
In file included from internal.c:1:
./internal/arg.c:66:41: warning: incompatible pointer types passing 'sexp **' (aka 'struct SEXPREC **') to parameter of type 'SEXP *' (aka 'void **') [-Wincompatible-pointer-types]
  sexp* expr = KEEP(capture(sym, frame, &env));
                                        ^~~~
./lib/rlang.h:54:14: note: expanded from macro 'KEEP'
#define KEEP PROTECT
             ^
/Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/include/Rinternals.h:718:31: note: expanded from macro 'PROTECT'
#define PROTECT(s)      Rf_protect(s)
                                   ^
./internal/arg.c:8:45: note: passing argument to parameter 'arg_env' here
sexp* capture(sexp* sym, sexp* frame, SEXP* arg_env) {
                                            ^
1 warning generated.
gcc -I/Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/include -DNDEBUG -I./lib/  -O2 -I/cm/shared/apps/pcre/8.38/include -I/cm/shared/apps/bzip2/1.0.6/include -I/cm/shared/apps/xz/5.2.2/include -I/cm/shared/apps/curl/7.50.1/include -DFASTR   -fPIC  -O2 -I/cm/shared/apps/pcre/8.38/include -I/cm/shared/apps/bzip2/1.0.6/include -I/cm/shared/apps/xz/5.2.2/include -I/cm/shared/apps/curl/7.50.1/include  -DFASTR -c lib.c -o lib.o
gcc -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/cm/shared/apps/bzip2/1.0.6/lib -L/cm/shared/apps/xz/5.2.2/lib -L/cm/shared/apps/pcre/8.38/lib -L/cm/shared/apps/curl/7.50.1/lib -L/cm/shared/apps/gcc/4.9.1/lib64 -L/usr/local/Cellar/gcc@4.9/4.9.4/lib/gcc/4.9/ -L/usr/lib -o rlang.so capture.o export.o internal.o lib.o
ld: warning: directory not found for option '-L/cm/shared/apps/bzip2/1.0.6/lib'
ld: warning: directory not found for option '-L/cm/shared/apps/xz/5.2.2/lib'
ld: warning: directory not found for option '-L/cm/shared/apps/pcre/8.38/lib'
ld: warning: directory not found for option '-L/cm/shared/apps/curl/7.50.1/lib'
ld: warning: directory not found for option '-L/cm/shared/apps/gcc/4.9.1/lib64'
ld: warning: directory not found for option '-L/usr/local/Cellar/gcc@4.9/4.9.4/lib/gcc/4.9/'
installing to /Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/library/rlang/libs
** R
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** testing if installed package can be loaded
com.oracle.truffle.r.runtime.ffi.DLL$DLLException: unable to load shared object '/Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/library/rlang/libs/rlang.so'
  dlopen(/Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/library/rlang/libs/rlang.so, 6): Symbol not found: _Rf_any_duplicated3
  Referenced from: /Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/library/rlang/libs/rlang.so
  Expected in: flat namespace
 in /Users/sivaram.konanki/graalvm-0.32/Contents/Home/jre/languages/R/library/rlang/libs/rlang.so
	at com.oracle.truffle.r.runtime.ffi.DLL$LoadPackageDLLNode.doLoad(DLL.java:532)
	at com.oracle.truffle.r.runtime.ffi.DLL$LoadPackageDLLNode.execute(DLL.java:486)
	at com.oracle.truffle.r.nodes.builtin.base.DynLoadFunctions$DynLoad.doDynLoad(DynLoadFunctions.java:77)
	at com.oracle.truffle.r.nodes.builtin.base.DynLoadFunctionsFactory$DynLoadNodeGen.execute(DynLoadFunctionsFactory.java:46)
	at com.oracle.truffle.r.nodes.builtin.RBuiltinNode$Arg4.call(RBuiltinNode.java:213)
	at com.oracle.truffle.r.nodes.builtin.InternalNode$InternalCallNode.execute(InternalNode.java:217)
	at com.oracle.truffle.r.runtime.nodes.RNode.visibleExecute(RNode.java:58)
	at com.oracle.truffle.r.nodes.function.FunctionDefinitionNode.execute(FunctionDefinitionNode.java:291)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:245)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:234)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:224)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:209)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:192)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.callProxy(OptimizedDirectCallNode.java:81)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:65)
	at com.oracle.truffle.r.nodes.function.call.CallRFunctionNode.execute(CallRFunctionNode.java:59)
	at com.oracle.truffle.r.nodes.function.RCallNode$DispatchedCallNode.execute(RCallNode.java:1125)
	at com.oracle.truffle.r.nodes.function.RCallNode$FunctionDispatch.dispatch(RCallNode.java:850)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen$FunctionDispatchNodeGen.execute(RCallNodeGen.java:660)
	at com.oracle.truffle.r.nodes.function.RCallNode.call(RCallNode.java:253)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen.execute(RCallNodeGen.java:158)
	at com.oracle.truffle.r.nodes.control.IfNode.execute(IfNode.java:83)
	at com.oracle.truffle.r.nodes.access.WriteLocalFrameVariableNodeGen.execute_generic3(WriteLocalFrameVariableNodeGen.java:114)
	at com.oracle.truffle.r.nodes.access.WriteLocalFrameVariableNodeGen.execute(WriteLocalFrameVariableNodeGen.java:48)
	at com.oracle.truffle.r.nodes.access.WriteVariableSyntaxNode.voidExecute(WriteVariableSyntaxNode.java:60)
	at com.oracle.truffle.r.nodes.control.BlockNode.visibleExecute(BlockNode.java:88)
	at com.oracle.truffle.r.nodes.function.FunctionDefinitionNode.execute(FunctionDefinitionNode.java:291)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:245)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:234)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:224)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:209)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:192)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.callProxy(OptimizedDirectCallNode.java:81)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:65)
	at com.oracle.truffle.r.nodes.function.call.CallRFunctionNode.execute(CallRFunctionNode.java:59)
	at com.oracle.truffle.r.nodes.function.RCallNode$DispatchedCallNode.execute(RCallNode.java:1125)
	at com.oracle.truffle.r.nodes.function.RCallNode$FunctionDispatch.dispatch(RCallNode.java:850)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen$FunctionDispatchNodeGen.execute(RCallNodeGen.java:660)
	at com.oracle.truffle.r.nodes.function.RCallNode.call(RCallNode.java:253)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen.execute(RCallNodeGen.java:158)
	at com.oracle.truffle.r.nodes.access.WriteLocalFrameVariableNodeGen.execute_generic3(WriteLocalFrameVariableNodeGen.java:114)
	at com.oracle.truffle.r.nodes.access.WriteLocalFrameVariableNodeGen.execute(WriteLocalFrameVariableNodeGen.java:48)
	at com.oracle.truffle.r.nodes.control.ReplacementNode$ReplacementWithRhsNode.voidExecute(ReplacementNode.java:214)
	at com.oracle.truffle.r.nodes.control.BlockNode.voidExecute(BlockNode.java:72)
	at com.oracle.truffle.r.nodes.control.ForNode$AbstractIndexRepeatingNode.executeRepeating(ForNode.java:286)
	at org.graalvm.compiler.truffle.runtime.OptimizedOSRLoopNode.profilingLoop(OptimizedOSRLoopNode.java:139)
	at org.graalvm.compiler.truffle.runtime.OptimizedOSRLoopNode.executeLoop(OptimizedOSRLoopNode.java:116)
	at com.oracle.graal.enterprise.V.d$b.executeLoop(stripped:157)
	at com.oracle.truffle.r.nodes.control.ForNode.iterate(ForNode.java:95)
	at com.oracle.truffle.r.nodes.control.ForNodeGen.execute(ForNodeGen.java:53)
	at com.oracle.truffle.r.runtime.nodes.RNode.voidExecute(RNode.java:50)
	at com.oracle.truffle.r.nodes.control.BlockNode.visibleExecute(BlockNode.java:88)
	at com.oracle.truffle.r.nodes.control.IfNode.visibleExecute(IfNode.java:96)
	at com.oracle.truffle.r.nodes.control.BlockNode.visibleExecute(BlockNode.java:90)
	at com.oracle.truffle.r.nodes.function.FunctionDefinitionNode.execute(FunctionDefinitionNode.java:291)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:245)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:234)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:224)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:209)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:192)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.callProxy(OptimizedDirectCallNode.java:81)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:65)
	at com.oracle.truffle.r.nodes.function.call.CallRFunctionNode.execute(CallRFunctionNode.java:59)
	at com.oracle.truffle.r.nodes.function.RCallNode$DispatchedCallNode.execute(RCallNode.java:1125)
	at com.oracle.truffle.r.nodes.function.RCallNode$FunctionDispatch.dispatch(RCallNode.java:850)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen$FunctionDispatchNodeGen.execute(RCallNodeGen.java:660)
	at com.oracle.truffle.r.nodes.function.RCallNode.call(RCallNode.java:253)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen.execute(RCallNodeGen.java:158)
	at com.oracle.truffle.r.nodes.access.WriteLocalFrameVariableNodeGen.execute_generic3(WriteLocalFrameVariableNodeGen.java:114)
	at com.oracle.truffle.r.nodes.access.WriteLocalFrameVariableNodeGen.execute(WriteLocalFrameVariableNodeGen.java:48)
	at com.oracle.truffle.r.nodes.access.WriteVariableSyntaxNode.voidExecute(WriteVariableSyntaxNode.java:60)
	at com.oracle.truffle.r.nodes.control.BlockNode.visibleExecute(BlockNode.java:88)
	at com.oracle.truffle.r.nodes.function.WrapArgumentBaseNode.visibleExecute(WrapArgumentBaseNode.java:63)
	at com.oracle.truffle.r.engine.REngine$AnonymousRootNode.execute(REngine.java:614)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:245)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:234)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:224)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:209)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.call(OptimizedCallTarget.java:185)
	at com.oracle.truffle.r.runtime.data.Closure.eval(Closure.java:119)
	at com.oracle.truffle.r.nodes.InlineCacheNode.evalPromise(InlineCacheNode.java:94)
	at com.oracle.truffle.r.nodes.InlineCacheNode.doGeneric(InlineCacheNode.java:75)
	at com.oracle.truffle.r.nodes.InlineCacheNodeGen.execute(InlineCacheNodeGen.java:42)
	at com.oracle.truffle.r.nodes.function.PromiseHelperNode.generateValueDefault(PromiseHelperNode.java:187)
	at com.oracle.truffle.r.nodes.function.PromiseHelperNode.evaluate(PromiseHelperNode.java:169)
	at com.oracle.truffle.r.nodes.function.PromiseHelperNode.generateValueNonDefault(PromiseHelperNode.java:208)
	at com.oracle.truffle.r.nodes.function.PromiseHelperNode.evaluate(PromiseHelperNode.java:171)
	at com.oracle.truffle.r.nodes.access.variables.ReadVariableNode.executeInternal(ReadVariableNode.java:267)
	at com.oracle.truffle.r.nodes.access.variables.LookupNode.visibleExecute(ReadVariableNode.java:112)
	at com.oracle.truffle.r.nodes.function.WrapArgumentBaseNode.visibleExecute(WrapArgumentBaseNode.java:63)
	at com.oracle.truffle.r.nodes.InlineCacheNode.doCached(InlineCacheNode.java:70)
	at com.oracle.truffle.r.nodes.InlineCacheNodeGen.execute(InlineCacheNodeGen.java:36)
	at com.oracle.truffle.r.nodes.function.PromiseHelperNode.generateValueDefault(PromiseHelperNode.java:187)
	at com.oracle.truffle.r.nodes.function.PromiseHelperNode.generateValueNonDefault(PromiseHelperNode.java:223)
	at com.oracle.truffle.r.nodes.function.PromiseHelperNode.evaluate(PromiseHelperNode.java:171)
	at com.oracle.truffle.r.nodes.access.variables.LocalReadVariableNode.execute(LocalReadVariableNode.java:151)
	at com.oracle.truffle.r.nodes.access.variables.LocalReadVariableNode.execute(LocalReadVariableNode.java:82)
	at com.oracle.truffle.r.nodes.function.PeekLocalVariableNode.execute(PeekLocalVariableNode.java:50)
	at com.oracle.truffle.r.nodes.function.PeekLocalVariableNode.visibleExecute(PeekLocalVariableNode.java:60)
	at com.oracle.truffle.r.nodes.builtin.base.ReturnSpecial.execute(Return.java:58)
	at com.oracle.truffle.r.nodes.function.RCallSpecialNode.execute(RCallSpecialNode.java:301)
	at com.oracle.truffle.r.nodes.function.RCallSpecialNode.visibleExecute(RCallSpecialNode.java:373)
	at com.oracle.truffle.r.nodes.function.WrapArgumentBaseNode.visibleExecute(WrapArgumentBaseNode.java:63)
	at com.oracle.truffle.r.nodes.InlineCacheNode.doCached(InlineCacheNode.java:70)
	at com.oracle.truffle.r.nodes.InlineCacheNodeGen.execute(InlineCacheNodeGen.java:36)
	at com.oracle.truffle.r.nodes.function.PromiseHelperNode.generateValueDefault(PromiseHelperNode.java:187)
	at com.oracle.truffle.r.nodes.function.PromiseHelperNode.evaluate(PromiseHelperNode.java:169)
	at com.oracle.truffle.r.nodes.access.variables.ReadVariableNode.executeInternal(ReadVariableNode.java:267)
	at com.oracle.truffle.r.nodes.access.variables.LookupNode.visibleExecute(ReadVariableNode.java:112)
	at com.oracle.truffle.r.nodes.control.BlockNode.visibleExecute(BlockNode.java:90)
	at com.oracle.truffle.r.nodes.function.FunctionDefinitionNode.execute(FunctionDefinitionNode.java:291)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:245)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:234)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:224)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:209)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:192)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.callProxy(OptimizedDirectCallNode.java:81)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:65)
	at com.oracle.truffle.r.nodes.function.call.CallRFunctionNode.execute(CallRFunctionNode.java:59)
	at com.oracle.truffle.r.nodes.function.RCallNode$DispatchedCallNode.execute(RCallNode.java:1125)
	at com.oracle.truffle.r.nodes.function.RCallNode$FunctionDispatch.dispatch(RCallNode.java:850)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen$FunctionDispatchNodeGen.execute(RCallNodeGen.java:660)
	at com.oracle.truffle.r.nodes.function.RCallNode.call(RCallNode.java:253)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen.execute(RCallNodeGen.java:158)
	at com.oracle.truffle.r.nodes.access.WriteLocalFrameVariableNodeGen.execute_generic3(WriteLocalFrameVariableNodeGen.java:114)
	at com.oracle.truffle.r.nodes.access.WriteLocalFrameVariableNodeGen.execute(WriteLocalFrameVariableNodeGen.java:48)
	at com.oracle.truffle.r.nodes.access.WriteVariableSyntaxNode.voidExecute(WriteVariableSyntaxNode.java:60)
	at com.oracle.truffle.r.nodes.control.BlockNode.visibleExecute(BlockNode.java:88)
	at com.oracle.truffle.r.nodes.function.FunctionDefinitionNode.execute(FunctionDefinitionNode.java:291)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:245)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:234)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:224)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:209)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:192)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.callProxy(OptimizedDirectCallNode.java:81)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:65)
	at com.oracle.truffle.r.nodes.function.call.CallRFunctionNode.execute(CallRFunctionNode.java:59)
	at com.oracle.truffle.r.nodes.function.RCallNode$DispatchedCallNode.execute(RCallNode.java:1125)
	at com.oracle.truffle.r.nodes.function.RCallNode$FunctionDispatch.dispatch(RCallNode.java:850)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen$FunctionDispatchNodeGen.execute(RCallNodeGen.java:660)
	at com.oracle.truffle.r.nodes.function.RCallNode.call(RCallNode.java:253)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen.execute(RCallNodeGen.java:158)
	at com.oracle.truffle.r.runtime.nodes.RNode.visibleExecute(RNode.java:58)
	at com.oracle.truffle.r.nodes.control.IfNode.visibleExecute(IfNode.java:93)
	at com.oracle.truffle.r.nodes.control.IfNode.visibleExecute(IfNode.java:96)
	at com.oracle.truffle.r.nodes.control.BlockNode.visibleExecute(BlockNode.java:90)
	at com.oracle.truffle.r.nodes.function.FunctionDefinitionNode.execute(FunctionDefinitionNode.java:291)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:245)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:234)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:224)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:209)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:192)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.callProxy(OptimizedDirectCallNode.java:81)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:65)
	at com.oracle.truffle.r.nodes.function.call.CallRFunctionNode.execute(CallRFunctionNode.java:59)
	at com.oracle.truffle.r.nodes.function.RCallNode$DispatchedCallNode.execute(RCallNode.java:1125)
	at com.oracle.truffle.r.nodes.function.RCallNode$FunctionDispatch.dispatch(RCallNode.java:850)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen$FunctionDispatchNodeGen.execute(RCallNodeGen.java:660)
	at com.oracle.truffle.r.nodes.function.RCallNode.call(RCallNode.java:253)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen.execute(RCallNodeGen.java:158)
	at com.oracle.truffle.r.runtime.nodes.RNode.visibleExecute(RNode.java:58)
	at com.oracle.truffle.r.nodes.control.BlockNode.visibleExecute(BlockNode.java:90)
	at com.oracle.truffle.r.nodes.function.FunctionDefinitionNode.execute(FunctionDefinitionNode.java:291)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callProxy(OptimizedCallTarget.java:245)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callRoot(OptimizedCallTarget.java:234)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:224)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.doInvoke(OptimizedCallTarget.java:209)
	at org.graalvm.compiler.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:192)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.callProxy(OptimizedDirectCallNode.java:81)
	at org.graalvm.compiler.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:65)
	at com.oracle.truffle.r.nodes.function.call.CallRFunctionNode.execute(CallRFunctionNode.java:59)
	at com.oracle.truffle.r.nodes.function.RCallNode$DispatchedCallNode.execute(RCallNode.java:1125)
	at com.oracle.truffle.r.nodes.function.RCallNode$FunctionDispatch.dispatch(RCallNode.java:850)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen$FunctionDispatchNodeGen.execute(RCallNodeGen.java:660)
	at com.oracle.truffle.r.nodes.function.RCallNode.call(RCallNode.java:253)
	at com.oracle.truffle.r.nodes.function.RCallNodeGen.execute(RCallNodeGen.java:158)
	at com.oracle.truffle.r.nodes.access.WriteLocalFrameVariableNodeGen.execute_generic3(WriteLocalFrameVariableNodeGen.java:114)
	at com.oracle.truffle.r.nodes.access.WriteLocalFrameVariableNodeGen.execute(WriteLocalFrameVariableNodeGen.java:48)
	at com.oracle.truffle.r.nodes.access.WriteVariableSyntaxNode.voidExecute(WriteVariableSyntaxNode.java:60)
	at com.oracle.truffle.r.nodes.control.BlockNode.voidExecute(BlockNode.java:72)
	at com.oracle.truffle.r.nodes.control.IfNode.voidExecute(IfNode.java:69)
	at com.oracle.truffle.r.nodes.control.BlockNode.voidExecute(BlockNode.java:72)
	at com.oracle.truffle.r.nodes.control.IfNode.voidExecute(IfNode.java:69)
	at com.oracle.truffle.r.nodes.control.BlockNode.voidExecute(BlockNode.java:72)
	at com.oracle.truffle.r.nodes.control.IfNode.voidExecute(IfNode.java:69)
	at com.oracle.truffle.r.nodes.control.BlockNode.visibleExecute(BlockNode.java:88)
	at com.oracle.truffle.r.nodes.function.FunctionDefinitionNode.execute(FunctionDefinitionNode.java:291)
```

**Comment by steve-s** [comment]
GraalVM version 0.32 unfortunately does not support the latest `rlang` package version (0.2), which was released just before 0.32 release which didn't leave us enough time to test it. The soon to be released 0.33 will support the newest `rlang` and `dplyr` should install with it fine. Workaround for now could be using the `checkpoint` package[1] to "rewind the time" back to say Feb 19 and then try installing dplyr (which should pull rlang version 0.1.6).

**Comment by sivakon** [comment]
I installed `dplyr` from this command.
`install.packages("dplyr", repos="https://cran.microsoft.com/snapshot/2018-02-02/")`

`sleepData <- select(msleep, name, sleep_total)`
this gives me the error
`not implemented: unexpected type: RLanguage(rep=RCallNodeGen@6e80de85) is RLanguage instead of RPairList (see fastr_errors.log)`

**Comment by steve-s** [comment]
This problem should be also solved in version 0.33. However, there is another problem with the code snippet you posted. We're working on making sure that FastR can run the most popular CRAN packages including `dplyr`, it is work in progress and we expect to be done in 1-3 months.

Thank you for reporting this. I've created an issue in the FastR repo [1] that you can watch and I will close this issue.

[1] https://github.com/oracle/fastr/issues/13
